Backend.c

```diff
@@ -1,130 +0,0 @@
-#include <stdio.h>
-#include <stdlib.h>
-#include <stdbool.h>
-#include <string.h>
```

Comment:
Now I can add comments to every content in your files.

Reply:
yey